dataset/src/inference/openai.ts

```diff
@@ -7,7 +7,7 @@ type OpenRouterError = {
   message: string;
 };
 
-export class OpenAI extends InferenceProvider {
+export class OpenAIInferenceProvider extends InferenceProvider {
```

Comment:
変だからCMUdict→CmuDictみたくこうしちゃっても良いのかもしれない
```suggestion
export class OpenaiInferenceProvider extends InferenceProvider {
```